ZWaveLib/Utility/Utility.cs

```diff
@@ -121,23 +113,6 @@ public static byte[] ExtractRoutingFromBitMask(byte[] receivedMessage)
             }
             return routingInfo.ToArray();
         }
-
-        public static void DebugLog(DebugMessageType dtype, string message)
```

Comment:
For that, the NLog configuration can be changed to colored console output (https://github.com/nlog/NLog/wiki/ColoredConsole-target). I would keep all logging calls using NLog so all messages from all libraries and code can be controlled from one place and in a consistent manner.